CHANGELOG.asciidoc

```diff
@@ -34,7 +34,12 @@ This is not a breaking change, so former
 <<config-disable-instrumentations,`disable_instrumentation`>> configuration containing the `incubating` tag will still be respected - {pull}1123[#1123]
 * Add a `--without-emulated-attach` option for runtime attachment to allow disabling this feature as a workaround.
 * Add workaround for JDK bug JDK-8236039 with TLS 1.3 {pull}1149[#1149]
-* Add log level `OFF` to silence agent logging
+* Logging
+** Add log level `OFF` to silence agent logging
+** Log files are now rotated after they reach <<config-log-file-max-size>>.
+  There will always be one history file `${log_file}.1`.
+** Add <<config-log-format-sout>> and <<config-log-format-file>> with the options `PLAIN_TEXT` and `JSON`.
```

Comment:
I like `sout` better because it's not only about console logging. Standard output is the standard log mechanism in docker containers. Docker picks up the standard output and writes that to a special log directory.